### ISSUE TITLE
MCW folder not present when initally logged into VM

### ISSUE THREAD

**Issue by richross** [microsoft/MCW-Modern-cloud-apps#35]
Hi - using this workshop as part of a customer lab.  One of the attendees created the VM and did not see the MCW folder after logging into the VM.  It took less than 5 minutes but the folder showed up on the c:\drive.  Is this expected behavior?
Great lab by the way!
Thanks!

**Comment by kylebunting** [comment]
@richross, the files should _typically_ be there when users log in. However, looking at the script that runs when the VM is created for this lab, it downloads and installs SSMS, which in my experience can sometimes take 10 or 15 minutes to complete, before it downloads and unzips the lab files. So, it is entirely possible that a user logging into the VM as soon as it is provisioned, but before the configuration script to completed running, could result in what you describe. 

The way provisioning works when using an ARM template, is the VM will be created, and once that is complete, a separate task to run the configuration script is started.

**Comment by richross** [comment]
I can add this in as a note and do a PR if that would be helpful.

**Comment by kylebunting** [comment]
I'll take care of adding the note, but thank you.

**Comment by kylebunting** [comment]
@richross, I just added an image and note about the additional deployment task, and that users should wait for that task to complete before attempting to log into the LabVM. Closing this issue.
